src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -109,6 +109,14 @@ public function getFilters()
         ];
     }
 
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('canonicalize_locale_for_moment', [$this, 'getCanonicalizedLocaleForMoment'], ['needs_context' => true]),
```

Comment:
Please use the new, namespaced version. See https://github.com/rectorphp/rector/blob/master/src/config/level/twig/underscore-to-namespace.yml